.github/workflows/tag-and-release.yml

```diff
@@ -32,10 +32,23 @@ jobs:
         # Must use a temporary file or it loses the formatting
         VERSION=${{steps.get_version.outputs.version}}; awk "/## \[$VERSION\]/{flag=1;next}/## \[/{flag=0}flag" CHANGELOG.md > REL-BODY.md
         
+    - name: Create Relase TAR
+      id: create_tar
+      run: |
+        mv aws-connect.sh aws-connect
+        tar cvzf ./aws-connect-${{steps.get_version.outputs.version}}.tar.gz aws-connect* README.md
+  
+    - name: Generate Checksum
+      id: generate_checksum
+      run: |
+        checksum=$(sha256sum ./aws-connect-${{steps.get_version.outputs.version}}.tar.gz | awk '{print $1}')
+        echo "${checksum}" > sha256.sum
+        echo "SHA256: ${checksum}" >> REL-BODY.md
+
     - name: Create Release
       uses: ncipollo/release-action@v1
       with:
         tag: ${{steps.get_version.outputs.version_tag}}
-        artifacts: "*.gem, CHANGELOG.md"
+        artifacts: "*.gem, CHANGELOG.md, sha256.sum, aws-connect*.tar.gz"
```

Comment:
This is good...but don't forget when you update the TAP that the url will now change...